src/legacy/ui/public/agg_types/controls/ip_range_type.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+
+import { EuiButton, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { AggParamEditorProps } from '../../vis/editors/default';
+
+enum IpRangeTypes {
+  MASK = 'mask',
+  FROM_TO = 'fromTo',
+}
+
+function IpRangeTypeParamEditor({ value, setValue }: AggParamEditorProps<IpRangeTypes>) {
+  const useFromToLabel = i18n.translate('common.ui.aggTypes.ipRanges.useFromToButtonLabel', {
+    defaultMessage: 'Use From/To',
+  });
+  const useCidrMasksLabel = i18n.translate('common.ui.aggTypes.ipRanges.useCidrMasksButtonLabel', {
+    defaultMessage: 'Use CIDR Masks',
+  });
+
+  const onClick = () => {
+    setValue(value === IpRangeTypes.MASK ? IpRangeTypes.FROM_TO : IpRangeTypes.MASK);
+  };
+
+  return (
+    <>
+      <EuiButton onClick={onClick}>
```

Comment:
Can you change this to us a `isFullWidth` style EuiButtonGroup (found towards the bottom of this page: https://elastic.github.io/eui/#/navigation/button). So that both options are visible?

It also looks better to remove the label (which would then be duplicative of the button label) like so:

<img width="293" alt="Screen Shot 2019-05-28 at 13 51 59 PM" src="https://user-images.githubusercontent.com/549577/58500374-04d5d480-8150-11e9-9c24-92b7a51ffac4.png">

You'd have to alter the aria-labelling by just adding the label as `aria-label` to the inputs.

Reply:
I noticed that not all buttons in `EuiButtonGroup` are accessible via keyboard. It seems like bug, so I created an issue: https://github.com/elastic/eui/issues/1979

Reply:
It wasn't the issue, radio operates using space, my bad.